### ISSUE TITLE
[BUG] Certain uploads cause RC to crash

### ISSUE THREAD

**Issue by mickymiek** [RocketChat/Rocket.Chat#14944]
### Description:

We noticed our RocketChat instances were crashing. We managed to trace the issue to a user that was uploading an SVG file. Could be related to [this issue](https://github.com/lovell/sharp/issues/1724).

### Steps to reproduce:

1. wget this file _(file omitted to prevent attacks)_
2. Rename it to something.svg
3. Upload it to RocketChat

### Expected behavior:

RocketChat shouldn't crash when uploading an SVG file.

### Actual behavior:

RocketChat crashes when the file is uploaded

### Server Setup Information:

- Version of Rocket.Chat Server: 1.2.1
- Operating System: Debian (CoreOS host)
- Deployment Method: Docker
- Number of Running Instances: 1
- DB Replicaset Oplog: On
- NodeJS Version: 8
- MongoDB Version: 4.0.10

### Additional context
When the lines below are removed from the file the upload goes fine.
```
<mask id="mask-2" fill="white">
    <use xlink:href="#path-1"></use>
</mask>
```

### Relevant logs:

```
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', libcore/option.rs:345:21
```

**Comment by snoopotic** [comment]
Someone is attacking us with that stupid :/

Any Idea to fix that?

**Comment by sampaiodiego** [comment]
thank you guys very much for your efforts.. I've omitted all information that could lead to potential attacks.. the issue is fixed and will be release on 1.3.0
